### DISCUSSION TITLE
不知道什么时候才能增加活跃橙的获取方式

### DISCUSSION THREAD

**Discussion by misividkoukou** [Meloong-Git/PCL#1851] [讨论]
总觉得现在活跃橙获取方式太单一了，弄得抢bug就像抢钱一样（
或者龙猫多来点bug也行
ε=ε=ε=┏(゜ロ゜;)┛

**Comment by WForst-Breeze** [comment]
应该不会考虑，活跃橙主要还是为了鼓励用户反馈问题或给 PCL2 做贡献的一种激励方式。其实目前获取活跃橙的途径已经不少了🤔就算不提交 Bug，只要会一点自定义主页和 XAML 格式的语法也可以很容易地为 PCL2 做出贡献并获得到活跃橙

**Reply by iPodtouch6557** [reply]
我正在写一个discussion，就是有一个部分关于活跃橙

**Comment by Chiloven945** [comment]
真正希望使MC社区以及PCL更好的玩家是不会去在意活跃橙的
